packages/starlight/components/MobileMenuFooter.astro

```diff
@@ -1,14 +1,21 @@
 ---
-import { LanguageSelect, ThemeSelect } from 'virtual:starlight/components';
+import { LanguageSelect, ThemeSelect, SocialIcons } from 'virtual:starlight/components';
 import type { Props } from '../props';
 ---
 
 <div class="mobile-preferences sl-flex">
+	<div class="sl-flex social-icons">
+		<SocialIcons {...Astro.props} />
+	</div>
 	<ThemeSelect {...Astro.props} />
 	<LanguageSelect {...Astro.props} />
 </div>
 
 <style>
+	.social-icons {
+		gap: 1rem;
+		align-items: center;
+	}
```

Comment:
We should probably also hide this wrapper element when there are no icons so that it only pushes the select menus to the right when there are actually some icons:

```suggestion
	}
	.social-icons:empty {
		display: none;
	}
```